migrations/sqls/20241116085102-initialize-up.sql

```diff
@@ -19,7 +22,7 @@ create table badges
 
 create table users
 (
-    id                text primary key,          -- using text as recommended
+    id                serial primary key,
```

Comment:
yeah uuids instead of these increasing numbers would make sense. 
Shall we make a ticket for using [uuidv6](https://datatracker.ietf.org/meeting/107/materials/slides-107-dispatch-uuid-v6-slides-brad-peabody-00.pdf)?
But for the users I actually would prefer to use unique readable user ids that users have to choose. Nicer to read API's that way. So I changed it in #35 to strings, but then I saw it cause extra migration work, so I just changed it back here the scope of these PR while keeping everthing that currently works working.

Reply:
Update: I created #44 for this